### PR TITLE
Feature/updated icons

### PR DESCRIPTION
**Pull request by cm0x4D**
Feel free to cherry pick if you do not like all new icons or features...

### CODE REVIEW

**Comment by Dolu1990** [comment]
73 commits <3
I have to try that now :)

**Comment by Dolu1990** [comment]
Very nice : D
Full agree, also realy like dataloger icon ! For me, it could be merged without cherry pick :)

**Comment by metc** [comment]
LGTM, very nice icons :star2:

File: .travis.yml [hunk 1]
```diff
@@ -14,4 +14,5 @@ cache:
     - $HOME/.gradle/wrapper/
 
 after_success:
-  - ./script/transifex.sh
\ No newline at end of file
+  - ./script/transifex.sh
+  - if [ "$TRAVIS_BRANCH" == "1.7.10-MNA" ]; then ./gradlew bintrayUpload; fi
```
Comment (metc) [inline .travis.yml:18]: Deploying snapshots is a good idea.
If we upload snapshots  from the `1.7.10-MNA` branch, we should work on features branches only, on not any more from the main branch.

**Comment by metc** [comment]
@cm0x4D I let you the honors to merge this PR, this is your work :star2:

**Comment by cm0x4D** [comment]
Damn. Now it can not be merged automatically anymore :-/

**Pull request merged**